### ISSUE TITLE
Please improve installation doc - unusable

### ISSUE THREAD

**Issue by hoagsie** [tlentz/d2modmaker#122]
I tried to follow the installation guide and found several unanswerable questions. As I won't use software I can't believe I've reliably setup, I can't use your app even if I think it's the coolest thing ever (it does look super neat). Here are the issues I ran into.

I have D2 from current Battle.net, not from a torrent site or from old installation disks. This auto installs as 1.14b and is already compatible with Windows 10 without messing with compatibility settings. Can I even use this app? I saw on a patch note that a bug was fixed for 1.14. Ok, that implies I can but it's a guess at this point as everything else refers to 1.13.

The instructions say to unzip to C:\D2. This isn't my install path but the rest of the guide sort of implies it is. I don't have a good mental model even for the file layout. If it's not supposed to be the same as the game installation directory, there are zero instructions on what to do after you've ran the mod maker.

I use PlugY, again out of the box. The installation guide seems to imply I'm using some set of custom command line args to get it work. I don't. So why does the guide specify an example? Are there some args I'm supposed to set? Does this not work if I don't? Again, another guess. If this is included for compatibility, it's not for compatibility to modern versions and is only confusing as presented.

I unzip to C:\D2, run the windows amd64 executable, and the react UI pops up. According to the guide since I'm on 1.14 and can't use the QoL settings. Well, the very first line on the react UI is Quality of Life. I'm back to questioning if I can even use this as it's not clear. Why aren't things separated to note what's reliable versus what works for older version? Even looking for the BaseMod download path has 113c in the file name. I'm assuming I can't even use that. OK...

As is, this app sounds like it was made during that time where even just a few years ago getting D2 working on modern systems was a chore or required using D2 in nonstandard ways compared to other software you'd install. That's not true anymore. As I said in my opening, going through the guide and trying to get something configured with it just doesn't work. It has assumptions that are no longer valid.

If I can't use it for 1.14 at all, please make that clear. If only parts aren't usable please make them more obvious. The basic functions aren't really even explained. After I click "run" what am I supposed to do then? How do I diagnose and debug issues?

I'm writing this only because there seems to be a level of care here so I'm trying to engage with it as a robust app and found it's not quite there. Cheers.

**Comment by irenusfer** [comment]
I have problems getting 0.6.0 to run as well, even when i downgrade to 1.13c

last version 0.5.4 worked for me perfectly on 1.14d so you can try to use that version.

**Comment by mightbemyname** [comment]
TLDR: How can I just *make a mod* with one or two changes and then just *play the game*? 

I'm trying to install this thing as well, and I just can't get over the abstruse nature of this tool and its documentation. Maybe it's only meant for people who are already neck-deep in the world of Diablo 2 mods.

- What do "Generator" and "Randomization" actually do? How would I know if I want to use them?
- Why is it an executable file that runs a local web site? 
- Why do I need multiple other mods like plugy?
- What does any of the shortcut language mean?
- I want to enable a couple of fun changes like "No minimum character level for items" and then just play the game. How?
- Does it really only work on 1.13c? What parts? All of it?
